### ISSUE TITLE
Als ontwikkelaar van de ZGW API's wil ik weten welk doel heeft het attribuut 'Kanaal' bij Verzoek heeft

### ISSUE THREAD

**Issue by hdksi** [VNG-Realisatie/gemma-zaken#1666]
...zodat ik kan bepalen of we het in de specs moeten houden.

Het attribuut 'Kanaal' bij Verzoek is volgens mij het gevolg van overerving uit de oude Klantinteracties API. Heeft dit attribuut waarde? Zie ook mijn opmerking hier: https://github.com/VNG-Realisatie/gemma-zaken/issues/1421#issuecomment-562160502.

**Comment by michielverhoef** [comment]
Aangezien een Verzoek nu los staat zou ik denken dat het attribuut communicatiekanaal zoals dat ook in de Zaak bekend is veel beter past: https://www.gemmaonline.nl/index.php/Rgbz_2.0/doc/attribuutsoort/zaak.communicatiekanaal 

_Het van een zaak beschikbaar zijn van het 'kanaal' is onder meer relevant om te kunnen bepalen op welke wijze met de initiator gecommuniceerd kan worden en om te achterhalen hoe de aanleiding voor de zaak ontvangen is. Een zaak kan bijvoorbeeld starten n.a.v. een ontvangen brief, een 'telefoontje', een ontvangen digitaal formulier (e-formulier), een e-mail etc. ._

Overigens staan hier 2 dingen in beschreven: het kanaal waarlangs een Zaak (of Verzoek) binnen is gekomen en het kanaal waarmee met de initator/verzoeker gecommuniceerd kan worden. Mijn voorkeur zou zijn hier twee attributen van te maken. Immers, over een Verzoek kan binnenkomen via een formulier op de website maar de initiator wordt ongetwijfeld liever teruggebeld/-gemaild/per brief op de hoogte gehouden etc.

**Comment by hdksi** [comment]
Ja, inderdaad: de RGBZ-definitie de toelichting sluiten niet op elkaar aan... Bestaan van het attribuut bij Zaak betekent in ieder geval dat er behoefte is aan het kunnen vastleggen van een voorkeurskanaal bij een Verzoek of Zaak. De originele vraag is van deze US is daarmee wat mij betreft beantwoord. Vraag is wel welke toegevoegde waarde het vastleggen van het kanaal waarlangs een Verzoek of Zaak is binnengekomen heeft.

NB: als je ervan uitgaat dat een Zaak altijd het gevolg is van een ontvangen Verzoek, dan zou het attribuut van op termijn voor Zaak kunnen vervallen...

**Comment by michielverhoef** [comment]
> NB: als je ervan uitgaat dat een Zaak altijd het gevolg is van een ontvangen Verzoek, dan zou het attribuut van op termijn voor Zaak kunnen vervallen...

In dat geval wordt de Verzoeken API wel heel hard gekoppeld aan de Zaken API en is het niet meer mogelijk om de Zaken API los van de Verzoeken API te implementeren.  Ik zou zelf een voorkeur hebben voor een apart attribuut bij een Zaak. Zeker wanneer bijvoorbeeld meerdere zaken uit een Verzoek komen en per zaaktype de voorkeurscommunicatie anders kan zijn (geen idee of dat ook echt zo kan zijn).

**Comment by Hugo-ter-Doest** [comment]
Het kanaal waarlangs een verzoek is ontvangen is interessant voor managementinformatie/business intelligence.
